packages/p2p/src/components/advertiser-page/advertiser-page.jsx

```diff
@@ -13,10 +13,13 @@ import { useStores } from 'Stores';
 import AdvertiserPageStats from './advertiser-page-stats.jsx';
 import AdvertiserPageAdverts from './advertiser-page-adverts.jsx';
 import TradeBadge from '../trade-badge/trade-badge.jsx';
+import BlockUserOverlay from './block-user/block-user-overlay';
+import BlockUserModal from 'Components/block-user/block-user-modal';
```

Comment:
Why is one absolute and the other one relative? Would this `BlockUserOverlay` work?

Reply:
Shouldn't `/block-user` be outside `/advertiser-page`?

Reply:
The block user overlay is only being used within advertiser page, thus the component is placed in a folder `block-user` inside advertiser page, while the block user modal is being used in other places, namely advertiser page and my profile page

Reply:
Can't we just keep overlay component (even if it's used in 1 place) inside `/block-user` and not `/advertiser-page/block-user`. Why have multiple `block-user` folders in different places? 🤔